### DISCUSSION TITLE
The official release 2.4.2 is marked as a "malware" in android in my Honor phone

### DISCUSSION THREAD

**Discussion by asmwarrior** [Nain57/Smart-AutoClicker#424] [Q&A]
I download the official release 2.4.2 apk file, after installed, there is a "!" around its icon, and my android phone(Honor, a phone produced by a Chinese company) says it is a "malware". To avoid this issue, I try install the beta version in this discussion form. Luckily it works fine.

One issue is that when started, the application will ask a "Accessibility" permission, so I have to enable this feature in the phone's setting. After that, this beta version works fine. But if I try to open this software next day, I still see the prompted that I need a "Accessibility" permission. 

So, there there any way to avoid such steps? It is always annoying if every time, I need to allow the "Accessibility" permission? This only happens I does not use this application for a long time, say several hours. My guess is that this application process is killed by the phone.

Another issue about using the beta version is that I see it can "select a small region of the screen"(for example a button), so that when we run the automation, it will first match that image. I see that if the button is very small, it is hard to tweak the selection rectangle. I mean it is hard to drag the edge of the selection rectangle when it is small.

Any way, many thinks for the tool.

**Comment by Aiadventures19** [comment]
well essentially it kinda is until u uninstall it.

**Reply by Nain57** [reply]
So an open source software, that anybody can read, is a malware ? 
Thx for making a fool of yourself, banned
